arviz/plots/dotplot.py

```diff
@@ -0,0 +1,229 @@
+    Everyday, Mobile Predictive Systems. DOI:https://doi.org/10.1145/2858036.2858558).
+
+    Parameters
+    ----------
+    values : array-like
+        Values to plot
+    binwidth : float
+        Width of the bin for drawing the dot plot.
+    dotsize : float
+        The size of the dots relative to the bin width. The default, 1, makes dots be
+        just about as wide as the bin width.
+    stackratio : float
+        The distance between the center of the dots in the same stack relative to the bin height.
+        The default, 1, makes dots in the same stack just touch each other.
+    point_interval : bool
+        Plots the point interval. Uses hdi_prob to plot the HDI interval
+    point_estimate : Optional[str]
+        Plot point estimate per variable. Values should be ‘mean’, ‘median’, ‘mode’ or None.
+        Defaults to ‘auto’ i.e. it falls back to default set in rcParams.
+    dotcolor : string
+        The color of the dots
+    intervalcolor : string
+        The color of the interval
+    linewidth : int
+        Line width throughout. If None it will be autoscaled based on figsize.
+    markersize : int
+        Markersize throughout. If None it will be autoscaled based on figsize.
+    markercolor: string
+        The color of the marker when plot_interval is True
+    marker: string
+        The shape of the marker. Valid for matplotlib backend
+        Defaults to "o"
+    hdi_prob : float
+        Valid only when point_interval is True. Plots HDI for chosen percentage of density.
+        Defaults to 0.94.
+    rotated : bool
+        Whether to rotate the dot plot by 90 degrees.
+    nquantiles : int
+        Number of quantiles to plot, used for quantile dot plots
+        Defaults to 50.
+    quartiles : bool
+        If True then the quartile interval will be plotted with the HDI.
+        Defaults to True.
+    figsize : tuple
+        Figure size. If None it will be defined automatically.
+    plot_kwargs : dict
+        Keywords passed for customizing the dots.
+    backend: str, optional
+        Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
+    ax : axes, optional
+        Matplotlib axes or bokeh figures.
+    show: bool, optional
+        Call backend show function.
+    backend_kwargs: bool, optional
+        These are kwargs specific to the backend being used. For additional documentation
+        check the plotting method of the backend.
+
+    Returns
+    -------
+    axes : matplotlib axes or bokeh figures
+
+    Examples
+    --------
+    Plot dot plot for a set of data points
+
+    .. plot::
+        :context: close-figs
+
+        >>> import numpy as np
+        >>> import arviz as az
+        >>> values = np.random.normal(0, 1, 500)
+        >>> az.plot_dist(values)
+
+    Manually adjust number of quantiles to plot
+
+    .. plot::
+        :context: close-figs
+
+        >>> az.plot_dist(values, nquantiles=100)
```

Comment:
Here and below:
```suggestion
        >>> az.plot_dot(values, nquantiles=100)
```

Reply:
yeah, my bad will change that

Reply:
@sethaxen `wilkinson_algorithm` function is in the `dotplot.py` file in plots directory so in which test file should I include the test for it?